blazor/treegrid/custom-toolbar.md

```diff
@@ -0,0 +1,119 @@
+---
+layout: post
+title: Custom Toolbar Items in Blazor TreeGrid Component | Syncfusion
+description: Learn how to create and use custom toolbar items in the Syncfusion Blazor TreeGrid, including templates, icons with text, dropdowns, and export actions.
```

Comment:
Check out here and learn more details about using custom toolbar items in the Syncfusion Blazor TreeGrid, including templates, icons with text, dropdowns, and export actions.